packages/commonwealth/client/scripts/helpers/abi_utils.ts

```diff
@@ -0,0 +1,17 @@
+// import { QueryTypes, Op }  from 'sequelize';
+// import { DB } from 'server/database';
+
+// const getContractAbi = async (models: DB, contractAddress: string) => {
```

Comment:
Empty file? Should also probably be changed to reflect Record<String, unknown> instead of just a string